3 Add system notifications

ateamcms opened the pull request.

irinaBerendeeva87 reviewed (changes requested):
### Checked: /implementation-notifications
❌ ✅
Statuses:
✅ Actief and Inactief

✅ Toggles

✅ Section "Deelnemers aanvraag en beoordeling"
✅ Section "Declaraties"
✅ Button "Verstuur een aangepast bericht"
✅  Button "Signature and corporate identity"
❌ opening sections lead to errors in console  and  empty screen:
Section "Deelnemers tegoeden":
- "Financieel-tegoed is geactiveerd"
- "Actie-tegoed is geactiveerd"
- "Aanbod-tegoed verlopen"

Section "Deelnemers reserveringen en transacties":
- "Aanbieder goedgekeurd voor het scannen van financiële tegoeden"
- "Aanbod-tegoed is geactiveerd"
- "Reservering aangemaakt"
- "Transactie van aanbod-tegoed"

Aanbieder aanvraag en beoordeling:
- "Aanbod geaccepteerd"
- "Reservering geannuleerd door aanvrager"
- " Aanbod geweigerd"
- " Aanbieder heeft een nieuw chatbericht ontvangen van de sponsor"

Aanbieder reserveringen en transacties:
- " Uitbetaling van een transactie"
- " Aanbod verlopen"
<img width="609" alt="Screenshot 2024-06-10 at 14 19 16" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/3fefd1eb-6de2-4bcf-b52b-b3998eae57c6">
<img width="604" alt="Screenshot 2024-06-10 at 14 18 44" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/ca88aded-efb5-4e5f-8be9-9c23d980be01">

Section "Deelnemers reserveringen en transacties"
❌ open sections lead to errors in console  and  empty screen: 
Aanbieder goedgekeurd voor het scannen van financiële tegoeden

ateamcms commented:
@irinaBerendeeva87 made fix

irinaBerendeeva87 reviewed (changes requested):
### Checked: /implementation-notifications

✅Opening sections

**Breadcrumbs**:
❌ After clicking "To look at " in any element in a section breadcrumb has  "Bijbetalingen" instead of "Systeemberichten"
<img width="275" alt="Screenshot 2024-06-11 at 11 10 29" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/8bc004ef-ed21-4abc-aab4-40ffc64556c1">
✅ Status "Gedeeltelijk" 
❌ Field "Kies een fonds" (in opened one of the settings)  should be empty with a placeholder  when the dropdown is opened
<img width="543" alt="Screenshot 2024-06-11 at 11 18 03" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/7231d585-5693-4e06-bc05-ad41fdd94903">
❌ sometimes switching toggles  and adding text lead to errors

<img width="1229" alt="Screenshot 2024-06-11 at 11 37 20" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/198862a2-79f3-4f35-b3ab-fe1e9f3e9957">
<img width="1244" alt="Screenshot 2024-06-11 at 11 37 10" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/f1b8a61a-2d2c-42e4-be3e-02200d5af265">
<img width="774" alt="Screenshot 2024-06-11 at 11 36 59" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/0fb0f14c-9dc6-4edc-8f50-0809945b41d6">

ateamcms commented:
@irinaBerendeeva87 made commit, can re-check

irinaBerendeeva87 reviewed (changes requested):
### Checked: /implementation-notifications
✅Opening sections
✅Breadcrumbs:
✅ Status "Gedeeltelijk"
✅ Field "Kies een fonds"

**Send a custom message**
❌ List of recipients  has pagination over an empty result
 /implementation-notifications/send
<img width="879" alt="Screenshot 2024-06-14 at 14 26 29" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/9409acee-4d21-41dd-9d34-d4daa79d971e">
✅ sorting
❌ Dropdown Kies een fonds has 8 fonds instead of 6
**Markdown**
❌  error when choosing _bold/italic style_ and adding space. 
<img width="680" alt="Screenshot 2024-06-14 at 14 43 49" src="https://github.com/teamforus/forus-frontend-react/assets/55141843/b597343a-e19a-41e7-826a-5336cc445f5a">

❌ same error when :
 1. add word with bold and italic  
 2. highlight the word
 3 click Remove Font Style
  4. then click add link

**Signature and corporate identity**
❌ Button "Reset" disappears after first click and don't delete img till empty state
 

https://github.com/teamforus/forus-frontend-react/assets/55141843/63005f57-d594-48b6-88d0-a9d583b623b0

irinaBerendeeva87 reviewed (changes requested):
### Retest: /implementation-notifications
**Send a custom message**
✅ pagination 
❌ Dropdown Kies een fonds has 5 fonds instead of 6
✅ bold/italic style

ateamcms commented:
Fixed switch between implementations

Dropdown Kies een fonds has 5 fonds instead of 6 - there was a bug in angular with filtering funds by implementation - it's not filtered, fixed in react

irinaBerendeeva87 reviewed (approved):
### Retest: /implementation-notifications
**Systeemberichten per webshop**
✅  Not to switch between 2 webshops
**Send a custom message**
✅ pagination
✅ Dropdown Kies een fonds has 5 fonds instead of 6
✅ bold/italic style

lexlog commented:
only one point during review:

on react version there is dropdown for funds appeared
![image](https://github.com/teamforus/forus-frontend-react/assets/8695780/6173b647-85a8-4737-8b69-14461be1dca6)

which is not present on angular
![image](https://github.com/teamforus/forus-frontend-react/assets/8695780/67e51ef7-d2bb-4a95-85c9-95b43a933cf6)

ateamcms commented:
@lexlog yes, that was a bug, fixed

lexlog commented:
all points are fixed, PR is approved for further merge
